backend/execution_cache/deployer/deploy-execution-cache.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is for deploying execution cache service to an existing cluster.
+# Prerequisite: config kubectl to talk to your cluster. See ref below: 
+# https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl
+
+set -ex
+
+echo "Start deploying execution cache to existing cluster:"
+
+NAMESPACE=${NAMESPACE_TO_WATCH:default}
```

Comment:
It seems to me that the default is not in effect. Might it be the following?

```suggestion
NAMESPACE=${NAMESPACE_TO_WATCH:-default}
```